datalake/src/main/java/com/sequenceiq/datalake/flow/create/event/StorageConsumptionCollectionSchedulingSuccessEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.datalake.flow.create.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.create.SdxCreateEvent;
+import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
+
+public class StorageConsumptionCollectionSchedulingSuccessEvent extends SdxEvent {
+
+    private final DetailedEnvironmentResponse detailedEnvironmentResponse;
+
+    @JsonCreator
+    public StorageConsumptionCollectionSchedulingSuccessEvent(
+            @JsonProperty("resourceId") Long sdxId,
+            @JsonProperty("userId") String userId,
+            @JsonProperty("detailedEnvironmentResponse") DetailedEnvironmentResponse detailedEnvironmentResponse) {
+        super(sdxId, userId);
+        this.detailedEnvironmentResponse = detailedEnvironmentResponse;
+    }
+
+    @Override
+    public String selector() {
+        return SdxCreateEvent.STORAGE_CONSUMPTION_COLLECTION_SCHEDULING_SUCCESS_EVENT.event();
```

Comment:
you can use `EventSelectorUtil`

Reply:
Done. The overriding impl has been removed, thus using the superclass logic.